picard/metadata.py

```diff
@@ -520,6 +520,22 @@ def __repr__(self):
     def __str__(self):
         return ("store: %r\ndeleted: %r\nimages: %r\nlength: %r" % (self._store, self.deleted_tags, [str(img) for img in self.images], self.length))
 
+    def _cover_art_description(self, forcount="no"):
```

Comment:
I think we should not place the helper function here. It is very UI related, and metadata itself is supposed to be primarily a data holding object.

Not 100% sure where to put this, but my suggestion would be to add it to the `Item` class (see https://github.com/metabrainz/picard/blob/master/picard/ui/item.py#L30), from which all those classes are derived. I would then change the function to accept an `images` parameter, so it can be called in the column methods as :

```
self.cover_art_description(self.metadata.images)
```

Another thing is that instead of `forcount` accepting a string it would be better as a boolean. I'd actually turn it around and call it `detailed`, so your function declaration looks like this:

```
def _cover_art_description(self, images, detailed=True):
```

You can then call it as:

```
self.cover_art_description(self.metadata.images, detailed=True)
```

or

```
self.cover_art_description(self.metadata.images, detailed=False)
```

which reads nicer.